type assertion panic in `windows/loader.go`: interface conversion: interface {} is *pefile.OptionalHeader32P, not *pefile.OptionalHeader32

Running `binee foo_32.exe` results in the following type assertion panic, as the type of the optional header is not `*pefile.OptionalHeader32`, but rather `*pefile.OptionalHeader32P`. See attached [foo_32.exe](https://github.com/carbonblack/binee/files/3877922/foo_32.tar.gz) sample.

```
$ binee foo_32.exe
panic: interface conversion: interface {} is *pefile.OptionalHeader32P, not *pefile.OptionalHeader32

goroutine 1 [running]:
github.com/carbonblack/binee/windows.(*WinEmulator).createLdrEntry(0xc000114900, 0xc000138100, 0x1, 0x9)
	/home/u/Desktop/binee/windows/loader.go:303 +0x3ec
github.com/carbonblack/binee/windows.(*WinEmulator).initPe(0xc000114900, 0xc000138000, 0x7ffec35b39b6, 0xa, 0x4, 0x4, 0xc000044640, 0x1, 0x1, 0xc000044600, ...)
	/home/u/Desktop/binee/windows/loader.go:826 +0xaad
github.com/carbonblack/binee/windows.New(0x7ffec35b39b6, 0xa, 0x4, 0x4, 0xc000044640, 0x1, 0x1, 0x0, 0x0, 0x0, ...)
	/home/u/Desktop/binee/windows/winemulator.go:288 +0x235b
main.main()
	/home/u/Desktop/binee/main.go:192 +0xa7b
```

--- a/windows/loader.go
+++ b/windows/loader.go
@@ -299,7 +299,16 @@ func (emu *WinEmulator) createLdrEntry(lpe *pefile.PeFile, index uint64) uint64 
 		LdrEntry.FullDllName.Length = uint16(nameLength)
 		LdrEntry.FullDllName.MaximumLength = uint16(nameLength)
 		LdrEntry.EntryPoint = uint32(lpe.EntryPoint())
-		LdrEntry.DllBase = uint32(lpe.OptionalHeader.(*pefile.OptionalHeader32).ImageBase)
+		var imageBase uint32
+		switch optHdr := lpe.OptionalHeader.(type) {
+		case *pefile.OptionalHeader32:
+			imageBase = uint32(optHdr.ImageBase)
+		case *pefile.OptionalHeader32P:
+			imageBase = uint32(optHdr.ImageBase)
+		default:
+			panic(fmt.Errorf("support for %T not yet implemented", lpe.OptionalHeader))
+		}
+		LdrEntry.DllBase = imageBase
 		LdrEntry.SizeOfImage = uint32(lpe.ImageSize)
 		LdrEntry.TlsIndex = uint16(index)
 

